### PR TITLE
add the steps to set up winpe for windows install

### PR DESCRIPTION
**Pull request by tannoa2**
@RackHD/corecommitters @uppalk1 @amymullins @zyoung51 @keedya

### CODE REVIEW

File: winpe/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Run the **Deployment and Imaging Tools Environment** application as administrator 
+You can do so by going to the **Start** menu, choose **All Programs** → **Windows Kits** → **Windows ADK**→ **Deployment and Imaging Tools Environment**
+- Copy the Windows PE files that you will be working with based on the hardware architecture that you are using. x86, amd64, or arm
+for example ```copype amd64 C:\WinPE_amd64_PS```
+
+###Add PowerShell to Windows PE
+- create a script with the bellow commands. ```notepad addingPowershell.bat```
+
+```
+Dism /Mount-Image /ImageFile:"C:\WinPE_amd64_PS\media\sources\boot.wim" /Index:1 /MountDir:"C:\WinPE_amd64_PS\mount"
```
Comment (zyoung51) [inline winpe/README.md]: Can you update the file paths to reference the Windows 10 ADK instead paths?

File: winpe/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#Editing the WINPE Operating System for windows installation
+
+
+###Download Windows ADK (Assessment and Deployment Kit)
+It is recommended that you use Windows ADK for windows 10
+https://developer.microsoft.com/en-us/windows/hardware/windows-assessment-deployment-kit
```
Comment (tannoa2) [inline winpe/README.md]: Hi @zyoung51 I have it over here. Is this the link you mentioned ?

Comment (zyoung51) [inline winpe/README.md]: The powershell commands below reference version 8.1.  Is that the right version number for the windows 10 ADK that you reference here?

File: winpe/README.md (hunk 1 above)
Comment (tannoa2) [inline winpe/README.md]: @zyoung51  Nice catch!!! 
you are right, it should be 10 not 8.1

**Pull request merged**